ui/component/app/view.jsx

```diff
@@ -295,6 +295,11 @@ function App(props: Props) {
     document.documentElement.setAttribute('theme', theme);
   }, [theme]);
 
+  useEffect(() => {
+    // $FlowFixMe
+    document.body.style.overflowY = currentModal ? 'hidden' : '';
+  }, [currentModal]);
```

Comment:
Seems like there are a lot of different types of modals in the app, one of which is the image viewer. So let me know if it is okay to do this for all modals.

Reply:
no it's okay. I can probably filter by just the view_image modal id
![image](https://user-images.githubusercontent.com/7473983/154688873-4259916b-3f77-4dbb-bfa7-a62b4d2c37da.png)

I just wasn't sure if it would be better to do it for everything.

For now I will just do it for the view_image modals

Reply:
okay I updated it so it's just the view modal. No worries!